vendor/bat-native-confirmations/src/bat/confirmations/internal/time_util.cc

```diff
@@ -0,0 +1,76 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
```

Comment:
It's a shame that we have to duplicate, but 🤷

Reply:
Totally agree, and I have raised https://github.com/brave/brave-browser/issues/9611